pom.xml

```diff
@@ -33,6 +33,16 @@
       <artifactId>appengine-api-1.0-sdk</artifactId>
       <version>1.9.59</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.endpoints</groupId>
+      <artifactId>endpoints-framework</artifactId>
+      <version>2.2.1</version>
+    </dependency>
```

Comment:
Please remove endpoints framework and just use simple servlet as in your starter projects. It is too heavy on configuration with no real benefit here.